backend/tasks/serializers.py

```
@@ -0,0 +1,88 @@
+        ]
+    
+    def create(self, validated_data):
+        # Add the employee from the request user
+        request = self.context.get('request')
+        if request and hasattr(request, 'user'):
+            from employees.models import Employee
+            try:
+                employee = Employee.objects.get(user=request.user)
+                validated_data['employee'] = employee
+            except Employee.DoesNotExist:
+                raise serializers.ValidationError("Employee profile not found for user")
```

Ruff (0.15.21)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)